pytorch_lightning/trainer/trainer.py

```diff
@@ -1773,7 +1773,6 @@ def reset_train_dataloader(self, model: Optional["pl.LightningModule"] = None) -
             self.train_dataloader,
             DataLoader,
             self._data_connector._prepare_dataloader,
-            shuffle=True,
```

Comment:
reviewer: this is removed from here and is being set inside `_prepare_dataloader`, because we can't check for its value here in case of a `CombinedLoader`.